### ISSUE TITLE
UX: revisit model search categories being filters

### ISSUE THREAD

**Issue by gfodor** [MozillaReality/Spoke#547]
Currently, the Sketchfab search uses the categories as search scopes/filters instead of top level search queries as is done in Hubs. This is useful if there are scenarios where you want to do a free form search within a specific category, but the downside is that users can often not realize the search is scoped. For example, if you go into the search box now, it defaults to featured, and if you type into the search box, you will get no results (and have to clear the "Featured" filter in order to get results.)

Consider an alternative design (like in Hubs) where the category/featured selections are just queries, not filters. In practice, it seems unlikely people will want to search within categories on Sketchfab and Poly since the search indexes are relatively small and the recall seems subjectively good without needing to filter.

**Comment by netpro2k** [comment]
Also ran into this, took me awhile to realize things were being filtered

**Comment by robertlong** [comment]
Yup, definitely agree. I think we need to think through the top bar of the library container entirely. There's a lot less vertical real estate for those categories that are in Hub's media search, but we can figure out how to make it work.

**Comment by gfodor** [comment]
I'd suggest we just change the UI for now to remove all filters after typing a query into the search box.

**Comment by robertlong** [comment]
I changed how this works in #762. I'm not sure if it addresses this issue or now.
